src/CMakeLists.txt

```
@@ -409,10 +416,20 @@ if(ENABLE_OPENCV)
       opencv_tracking
       openshot_protobuf
     )
-  set(HAVE_OPENCV TRUE CACHE BOOL "Building with OpenCV effects" FORCE)
-  mark_as_advanced(HAVE_OPENCV)
+    set(HAVE_OPENCV TRUE CACHE BOOL "Building with OpenCV effects" FORCE)
+    mark_as_advanced(HAVE_OPENCV)
+
+    # Keep track of OpenCV version, to embed in our version header
+    set(OPENCV_VERSION_STR "${OpenCV_VERSION}" CACHE STRING "OpenCV version linked with" FORCE)
+    mark_as_advanced(OPENCV_VERSION_STR)
+
+    # We may need to use Tracker as opencv::legacy::Tracker
+    if(OpenCV_VERSION VERSION_GREATER_EQUAL 4.5.2)
```

`VERSION_GREATER_EQUAL` is CMake 3.7+, and I've grown wary of making us dependent on such a "new" feature like that. Better to stick with `VERSION_GREATER`, which will be semantically equivalent in this context and has existed since before CMake 3.0.

```suggestion
    if(OpenCV_VERSION VERSION_GREATER "4.5.1")
```